src/Microsoft.Dnx.Host.CoreClr/DomainManager.cs

```diff
@@ -11,8 +12,24 @@
 [SecurityCritical]
 sealed class DomainManager
 {
+    [StructLayout(LayoutKind.Sequential)]
+    private unsafe struct NativeBootstrapperContext
+    {
+        public char* OperatingSystem;
+
+        public char* OsVersion;
+
+        public char* Architecture;
+
+        public uint NumProcessors;
```

Comment:
If that's the case, can we clarify the property name? `PhysicalProcessorCount`? (I'd also like to avoid using prefixes like `Num` that we don't use elsewhere, even though this is private ;))